lib/bai2/record.rb

```diff
@@ -30,6 +30,7 @@ class Record
 
     # Returns a time interval in seconds, to be added to the date
     ParseMilitaryTime = -> (v) do
+      v = '0000' if (v == '' || v == nil)
```

Comment:
If we want to be _fully_ up to spec, we'd actually have two anonymous functions (or a single function that returns another anonymous function) here `ParseRequiredMilitaryFile` (for the file header, would throw a `ParseError` if it got an empty string) and `ParseMilitaryTime`.

Reply:
Technically 2400 is the same as `0000` so this might as well be one just if condition including v = '9999'

Reply:
👍